tsconfig.json

```diff
@@ -18,10 +18,17 @@
     "node_modules",
     "playground/dist"
   ],
+  // It must include everything that Rollup will encounter,
+  // and preferably nothing else in order not to compile excess files
+  "include": [
+    "src/index.ts",
+    "src/**/*.d.ts",
+    "resources/content_blocking/selectors_tester.ts"
```

Comment:
`resources/content_blocking/selectors_tester.ts` doesn't exist in this project, so this line should be removed